scripts/restore

```diff
@@ -53,16 +50,19 @@ ynh_restore
 yunohost service add $app
 systemctl enable $app.timer --quiet
 systemctl start $app.timer
-#yunohost service add $app.timer
-#yunohost service enable $app.timer
-#yunohost service start $app.timer
```

Comment:
If those ain't needed anymore, let's remove them entirely instead of keeping them as comments...